patches/hash-base++readable-stream+3.6.0.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/node_modules/hash-base/node_modules/readable-stream/lib/_stream_readable.js b/node_modules/hash-base/node_modules/readable-stream/lib/_stream_readable.js
+index 192d451..bf3e60a 100644
+--- a/node_modules/hash-base/node_modules/readable-stream/lib/_stream_readable.js
++++ b/node_modules/hash-base/node_modules/readable-stream/lib/_stream_readable.js
+@@ -45,7 +45,7 @@ var Stream = require('./internal/streams/stream');
+ 
+ var Buffer = require('buffer').Buffer;
+ 
+-var OurUint8Array = global.Uint8Array || function () {};
```

Comment:
Why do we need to remove `global.Uint8Array` here?

Reply:
Otherwise there is an error "global is undefined", there should be a better fix for that. Some recommend to use some rollup polyfill, i tried bit somehow its not working.